package/utils/lua5.3/Makefile

```diff
@@ -40,8 +40,22 @@ define Package/lua5.3/Default/description
   programming, data-driven programming, and data description.
 endef
 
+define Package/liblua5.3
+$(call Package/lua53/Default)
+  SUBMENU:=
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE+= (libraries)
+endef
```

Comment:
Please keep `ABI_VERSION` there just like in the `package/utils/lua/Makefile`. It's required to trigger recompilation of all dependant software on every library ABI change.

Reply:
@rmilecki  - I've added this ABI_VERSION

Reply:
@jow- just pointed out https://abi-laboratory.pro/index.php?view=timeline&l=lua to me
It seems Lua keeps ABI compatibility for all 5.3.x releases, so "ABI_VERSION should be 5.3"

Reply:
@rmilecki  - updated ABI_VERSION to just 5.3 based on your request.